docs/tour/comments.md

```diff
@@ -1,6 +1,10 @@
 # Comments
 
-Comments in D2 begin with a hash.
+In D2 there are block comments and line comments.
```

Comment:
minor conciseness: `D2 has line comments and block comments"

(also ordered since line comments are explained first)